diff --git a/requirements.txt b/requirements.txt
index eee8ac7..9468ea1 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,3 @@
-enum34
 gmpy2
 zope.interface
 cryptography
diff --git a/setup.py b/setup.py
index 040c619..71a0776 100755
--- a/setup.py
+++ b/setup.py
@@ -31,6 +31,6 @@ setup(
 
     packages=['otr'],
     provides=['otr'],
-    install_requires=['gmpy2', 'zope.interface', 'application', 'cryptography']
+    install_requires=['gmpy2', 'zope.interface', 'cryptography']
 )
 
