diff --git a/resources/add_account.ui b/resources/add_account.ui
index 5b1b27a..702c31b 100644
--- a/resources/add_account.ui
+++ b/resources/add_account.ui
@@ -91,7 +91,14 @@
       <item>
        <widget class="QFrame" name="input_frame">
         <property name="styleSheet">
-         <string>QFrame#input_frame {
+         <string>
+* {
+	color: #000000;
+}
+QWidget {
+	background-color: transparent;
+}
+QFrame#input_frame {
 	border: 2px;
 	border-radius: 4px;
 	border-style: solid;
